libs/ngx-videogular/tsconfig.lib.json

```diff
@@ -32,8 +32,9 @@
     "annotateForClosureCompiler": true,
     "strictMetadataEmit": true,
     "flatModuleOutFile": "index.js",
-    "flatModuleId": "@videogular/ngx-videogular",
+    "flatModuleId": "@49ing/ngx-videogular",
     "skipTemplateCodegen": true,
     "enableResourceInlining": true,
+    "compilationMode": "partial"
```

Comment:
needed fix to avoid 

```
> @videogular/ngx-videogular@5.0.1 prepublishOnly
> node --eval "console.error('ERROR: Trying to publish a package that has been compiled by Ivy in full compilation mode. This is not allowed.\nPlease delete and rebuild the package with Ivy partial compilation mode, before attempting to publish.\n')" && exit 1

ERROR: Trying to publish a package that has been compiled by Ivy in full compilation mode. This is not allowed.
Please delete and rebuild the package with Ivy partial compilation mode, before attempting to publish.
```